shell/browser/electron_permission_manager.cc

```diff
@@ -142,6 +143,16 @@ void ElectronPermissionManager::SetBluetoothPairingHandler(
   bluetooth_pairing_handler_ = handler;
 }
 
+bool ElectronPermissionManager::IsGeolocationDisabledViaCommandLine() {
```

Comment:
```suggestion
// static
bool ElectronPermissionManager::IsGeolocationDisabledViaCommandLine() {
```

nit: static members should have comment `// static` in the implementation.